demisto_sdk/commands/lint/linter.py

```diff
@@ -60,7 +59,7 @@ def __init__(self, pack_dir: Path, content_repo: Path, req_3: list, req_2: list,
             "images": [],
             "python_version": 0,
             "env_vars": {},
-            "test": False,
+            "tests": [],
             "lint_files": [],
             "lint_unittest_files": [],
```

Comment:
You can remove `lint_unittest_files` here (and anywhere else if it's still left-over mentioned anywhere else in the code).

Reply:
Done